syft/workers/websocket_client.py

```diff
@@ -36,6 +38,14 @@ def __init__(
 
         super().__init__(hook, id, data, is_client_worker, log_msgs, verbose)
 
+    def search(self, *query):
+        # Prepare a message requesting the websocket server to search among its objects
+        message = (MSGTYPE.SEARCH, query[0])
```

Comment:
I just realized that this should probably send the whole `query`, not just the first element. Which would mean that on the receiving end, after deserializing, this would result in tuple of elements and would need to be passed to `search` with `*` in front. This would require some more changes in base.py, though. Hm...

Reply:
Agreed

Reply:
(changes to base.py for this purpose are ok)

Reply:
this is now addressed